src/modules/simulator/simulator_mavlink.cpp

```diff
@@ -544,6 +544,33 @@ void Simulator::poll_topics()
 	if (updated) {
 		orb_copy(ORB_ID(vehicle_status), _vehicle_status_sub, &_vehicle_status);
 	}
+
+	orb_check(_tune_control_sub, &updated);
+
+	if (updated) {
+
+		orb_copy(ORB_ID(tune_control), _tune_control_sub, &_tune_control);
+
+		_tunes.set_control(_tune_control);
+
+		mavlink_play_tune_t tune_msg = { };
+		mavlink_message_t message = { };
+
+		union {
+			struct {
+				unsigned int frequency;
+				unsigned int duration;
+				unsigned int silence_length;
+			};
+			uint8_t a[sizeof(tune_msg.tune)];
```

Comment:
@catch-twenty-two , I'm not a huge fan of single letter variables outside of index usage,... Any chance we could come up with a more descriptive name for `a`?

Reply:
this is gone now